tests/test_lab.py

```diff
@@ -114,7 +114,7 @@ def has_debug_params(self) -> bool:
     Command(("config", "init"), needs_config=False, should_fail=False),
     Command(("config", "show")),
     Command(("model",), needs_config=False, should_fail=False),
-    Command(("model", "chat")),
+    Command(("model", "chat"), ("--rag",)),
```

Comment:
Sorry, I'm not understanding and maybe this is my fault.. :)

Why are we adding `--rag` here? I know we need to test `Command(("model", "chat"), ("--rag",))`, but don't we also need to test the original use case of `Command(("model", "chat")`? If we add `--rag`, will the `Command(("model", "chat")` base use case be implicitly tested here? If yes, ignore my comment. If not, I think we need to ensure we're testing with and without `--rag`.

Reply:
I misunderstood how this parametrization was being done, good catch! Fixed.